drivers/java/test-resources/Hello.java

```diff
@@ -5,7 +5,7 @@
 
 public class Hello {
     public static void main(String[] args) {
-        System.out.println("Hello");
+        System.out.println("Hello, the current working directory is: " + System.getProperty("user.dir"));
```

Comment:
I'm not much of a Java developer, but do we need to have both the class file and the jar committed as blobs in the repo? Or can we get away with just the jar?

Reply:
I believe so. You can actually look at the jar contents and its essentially just pointing to the class file, which I believe contains the bytecode.